### PR TITLE
Attach acquisition metadata

### PR DESCRIPTION
**Pull request by eyshin05**

### CODE REVIEW

**Comment by junyoon** [comment]
Could you please remove all the commented codes?

File: src/generate_xml.py [hunk 1]
```diff
@@ -745,10 +750,10 @@ def populate_xml(datatype: str, id: int, filepath: str, conn: BlitzGateway,
         populate_screen(obj, ome, conn, hostname, metadata)
     elif datatype == 'Plate':
         populate_plate(obj, ome, conn, hostname, metadata)
-    if not barchive:
-        with open(filepath, 'w') as fp:
-            print(to_xml(ome), file=fp)
-            fp.close()
+    # if not barchive:
```
Comment (junyoon) [inline src/generate_xml.py]: please clean up the commented codes.

File: src/generate_xml.py [hunk 2]
```diff
@@ -370,10 +370,10 @@ def create_filepath_annotations(id: str, conn: BlitzGateway,
...
             anrefs.append(anref)
         else:
-            if fpaths:
-                f = fpaths[0]
-            else:
-                f = f'pixel_images/{clean_id}.tiff'
+            # if fpaths:
+            #     f = fpaths[0]
+            # else:
+            f = f'pixel_images/{clean_id}.tiff'
```
Comment (junyoon) [inline src/generate_xml.py:373]: please clean up the commented codes.

**Comment by eyshin05** [comment]
Thank you for your comment! I've made the changes you suggested. Please let me know if there's anything else you think needs to be done.

**Comment by junyoon** [comment]
Thanks for your updates.

**Pull request merged**